plugins/analysis/java-plugin/src/main/kotlin/arrow/meta/plugins/analysis/java/ast/JavaResolutionContext.kt

```diff
@@ -115,10 +115,10 @@ public class JavaResolutionContext(private val ctx: AnalysisContext) : Resolutio
     reportError(declaration, AnalysisMessages.UnsatBodyPost, msg)
 
   override fun reportInconsistentCallPost(expression: Element, msg: String): Unit =
-    reportError(expression, AnalysisMessages.InconsistentCallPost, msg)
+    reportWarning(expression, AnalysisMessages.InconsistentCallPost, msg)
```

Comment:
how come these are becoming warnings?

Reply:
This was in response to Kotlin: if you write a `TODO`, then the rest of the code is unreachable. However, you don't want to show all your `TODO`s as errors in the analysis, since that stops further steps in the compilation. But changing it (or making it configurable) is another option.